src/System.Windows.Forms/src/System/Windows/Forms/Label.cs

```diff
@@ -1270,7 +1270,7 @@ protected override void OnTextChanged(EventArgs e)
                 Invalidate();
             }
 
-            if (LiveSetting != AutomationLiveSetting.Off)
+            if (LiveSetting != AutomationLiveSetting.Off && IsAccessibilityObjectCreated)
```

Comment:
Switch the conditions:
```suggestion
            if (IsAccessibilityObjectCreated && LiveSetting != AutomationLiveSetting.Off)
```